extensions/components/com_redcore/site/helpers/translation.php

```diff
@@ -0,0 +1,167 @@
+<?php
```

Comment:
To avoid the redundancy of having identical files I assume? Yeah, that seems like a good solution.